testsuite/classlibrary/TestStream.sc

```diff
@@ -0,0 +1,42 @@
+TestStream : UnitTest {
+
+	test_funcStream {
+		var str, envir;
+
+		envir = Environment.new;
+		envir.use {
+			~count = 0;
+			str = FuncStream({ ~count = ~count + 1 }, { ~count = 0 });
+		};
+
+		this.assert(str.nextN(8).postln == (1..8), "FuncStream should call the nextFunc in original environment");
```

Comment:
Wouldn't `assertEquals`, without the `postln`, be a more canonical/readable way to do these?